Mars Onboarding/Hooks/LangAndSkillHooks.cs

```diff
@@ -0,0 +1,29 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Reqnroll;
+
+namespace Mars_Onboarding.Hooks
+{
+    [Binding]
+    public sealed class LangAndSkillHooks
+    {
+        private  IWebDriver? driver;
+
+        [BeforeScenario("@tag1")]
+        public void BeforeScenario(ScenarioContext scenarioContext)
+        {
+            driver = new ChromeDriver();
```

Comment:
you are initialising driver in a few places you should only initialise this once

Reply:
so public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }
and then in the hooks  create a common driver instance which calls the initialise method